tests/tests.cpp

```diff
@@ -15,12 +15,6 @@
 // For the implementation of the fixture see tests/windowFixture.hpp
 // --------------------------------------------
 
-TEST_F(Window, Basic)
-{
-	// Basic window is already tested in the fixture
-}
-
-
```

Comment:
This is testing the constructor

Reply:
In that case it shouldn't be removed, excuse me